src/TheThingsNetwork.cpp

```diff
@@ -7,9 +7,12 @@
 #define debugPrintLn(...) { if (debugStream) debugStream->println(__VA_ARGS__); }
 #define debugPrint(...) { if (debugStream) debugStream->print(__VA_ARGS__); }
 
-void TheThingsNetwork::init(Stream& modemStream, Stream& debugStream) {
-  this->modemStream = &modemStream;
-  this->debugStream = &debugStream;
+void TheThingsNetwork::init(Stream& modemStream) {
+  this->modemStream = &modemStream; 
+  if (!this->debugStream) {
+    Serial.begin(9600);
+    TheThingsNetwork::setDebugSerial(Serial);
+  }
```

Comment:
This is assuming that `Serial` is set on all platforms. I can imagine that there are Arduino boards that don't have a `Serial` defined.

Note that if the `debugStream` is not opened yet, this check also returns true. So without initializing the debug serial in the user sketch, it will always be reset to `Serial`, and `begin()` may be called twice.

Also, why calling functions as static functions all the time?